image_names.js

```diff
@@ -251,8 +253,8 @@ function getItems (unzippedFilesDir, itemsTexturesPath, itemMapping, version) {
     const texture = extractModel('item/' + model, unzippedFilesDir + '/assets/minecraft/models/')
     return {
       name: item.name,
-      model: model === null ? null : model.replace('item/', 'items/'),
-      texture: texture === null ? null : texture.replace('item/', 'items/')
+      model: !model ? null : model.replace('item/', 'items/'),
```

Comment:
can we explicitly handle all the "null" like cases ? it's a bit dangerous to rely on js rules for this

Reply:
these variables can be null or undefined depending on the version, the alternative is to write (model === null || model === undefined) which is a bit too verbose

Reply:
I'd better have either `model === null || model === undefined` or a isUndefined(model)
Writing !model means model === null || model === undefined || model === 0 || model === '' which I think is risky

Reply:
in this case, we don't want it to === 0 either (otherwise replace will fail) and if its '' we probably don't want it either ?
In any case 0 and '' are not possible values for texture or model, and if was the case, why would it be risky to ignore them (assuming that model and texture should be valid path) ?

Reply:
yeah I see you're point, in this case it makes sense, let's leave it as is